docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+datastore:
+  image: mongo:3.0.2
+  ports:
+    - "27017:27017"
+frontend:
+  build: .
+  command: python -u main.py
+  ports:
+    - "5000:5000"
+  volumes:
+      - .:/src
+  links:
```

Comment:
Docker recommends using user-defined networks instead of links to connect services. In that way, services can be reached by their name:

Check out the docker compose file from the workflow editor service for a similar example.

Reply:
```
version: '3'

services:

  workflow-editor-datastore:
    image: mongo:3.0.2
    ports:
      - "27017:27017"
    networks:
      - database

  editor:
    build: .
    ports:
      - "8080:8080"
    depends_on:
      - workflow-editor-datastore
    volumes:
      - .:/src
    networks:
      - database

networks:
  database:
 ```